src/dualscreen.tsx

```diff
@@ -294,45 +289,53 @@ export class DualScreen extends KalturaPlayer.core.BasePlugin {
-  //   const request = new RequestBuilder(headers);
-  //   request.service = 'baseEntry';
-  //   request.action = 'list';
-  //   request.method = 'POST';
-  //   request.tag = 'list';
-  //   request.params = {
-  //     filter: {objectType: 'KalturaBaseEntryFilter', parentEntryIdEqual: this._player.getMediaInfo().entryId},
-  //     responseProfile: {
-  //       type: ResponseProfileType.INCLUDE_FIELDS,
-  //       fields: 'id'
-  //     }
-  //   };
-  //   this._player.provider.getCustomData({requests: [request]}).then((response: any) => {
-  //     this._secondaryKalturaPlayer.loadMedia({entryId: response[0].data.objects[0].id});
-  //     this._setMode();
-  //     this._videoSyncManager = new VideoSyncManager(this.eventManager, this.player, this._secondaryKalturaPlayer, this.logger);
-  //   });
-  // }
+  private _getSecondaryMedia() {
+    this._player.provider
+      .doRequest([{loader: SecondaryMediaLoader, params: {parentEntryId: this._player.getMediaInfo().entryId}}])
+      .then((data: Map<string, any>) => {
+        if (data && data.has(SecondaryMediaLoader.id)) {
+          const secondaryMediaLoader = data.get(SecondaryMediaLoader.id);
+          if (secondaryMediaLoader && secondaryMediaLoader.response) {
+            if (secondaryMediaLoader.response.entryId === undefined) {
+              this.logger.error('Secondary entry id not found');
+            } else {
+              this._secondaryKalturaPlayer.loadMedia({entryId: secondaryMediaLoader.response?.entries[0]?.id});
+              this._videoSyncManager = new VideoSyncManager(this.eventManager, this.player, this._secondaryKalturaPlayer, this.logger);
+              this.eventManager.listen(this._secondaryKalturaPlayer, this.player.Event.FIRST_PLAYING, () => {
+                this.logger.debug('secondary player first playing - show dual mode');
+                // TODO: uncomment in next release
+                // this._secondaryKalturaPlayer.muted = true;
+                this._setMode();
+              });
+            }
+          }
+        }
+      })
+      .catch((e: any) => {
+        this.logger.error(e);
+      });
+  }
 
-  private _createChildPlayer() {
-    let childPlaceholder = document.createElement('div');
-    childPlaceholder.setAttribute('id', 'childPlaceholder');
-    childPlaceholder.style.width = '240px';
-    childPlaceholder.style.height = '135px';
-    childPlaceholder.hidden = true;
-    document.body.appendChild(childPlaceholder);
-    const childPlayerConfig = {
-      targetId: 'childPlaceholder',
+  private _createSecondaryPlayer() {
+    let secondaryPlaceholder = document.createElement('div');
+    secondaryPlaceholder.setAttribute('id', 'secondaryPlaceholder');
+    secondaryPlaceholder.style.width = '240px';
+    secondaryPlaceholder.style.height = '135px';
+    secondaryPlaceholder.hidden = true;
+    document.body.appendChild(secondaryPlaceholder);
+    const secondaryPlayerConfig = {
+      targetId: 'secondaryPlaceholder',
```

Comment:
add `disableUserCache: true` to not consider the user preferences

Reply:
done